.Renviron

```diff
@@ -1,2 +1,3 @@
 ARROW_S3=ON
 ARROW_WITH_SNAPPY=ON
+AWS_REGION=us-east-1
```

Comment:
This is present in the res repo but not this one. Not having it causes the SNS topic publish to fail.